lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -21,6 +21,13 @@ all_interfaces:
   get_context: ~
   get_value: '/^interface (.*)/'
 
+bfd_echo:
+  _exclude: [ios_xr]
+  kind: boolean
+  get_value: '/^(?:no )?bfd echo$/'
+  set_value: '<state> bfd echo'
+  default_value: true
```

Comment:
For now leaving it as it is. When developing the puppet provider code, will revisit this to see if there is a cleaner solution.